sklearn/utils/extmath.py

```diff
@@ -641,7 +642,8 @@ def make_nonnegative(X, min_value=0):
     return X
 
 
-def _batch_mean_variance_update(X, old_mean, old_variance, old_sample_count):
+def _batch_mean_variance_update(X,
+                                last_mean, last_variance, last_sample_count):
```

Comment:
I find this indentation unusual: please insert a new line only when needed, that is, just before the arg that would have caused an overflow otherwise.